app/src/main/java/com/cp/tasky/agenda/overview/domain/OverviewRepository.kt

```diff
@@ -0,0 +1,12 @@
+package com.cp.tasky.agenda.overview.domain
+
+import com.cp.tasky.agenda.overview.domain.model.LocalCacheOverview
+import com.cp.tasky.agenda.overview.domain.model.Overview
+
+interface OverviewRepository {
+    suspend fun getAgendaByDate(timeStamp: Long): Overview
```

Comment:
Have you worked with `LocalDateTime`/`ZonedDateTime` before instead of Long timestamps? Those are much more convenient and make it a lot easier to deal with dates and timezones.